client/src/components/ProfileView/ProfileView.js

```diff
@@ -45,13 +45,33 @@ export default function ProfileView() {
   const classes = useStyles();
   const [openFriendDialog, setOpenFriendDialog] = React.useState(false);
   const [openPollDialog, setOpenPollDialog] = React.useState(false);
+  const [myFriendsLists, setMyFriendsLists] = React.useState([]);
+  const [myPolls, setMyPolls] = React.useState([]);
+
+  React.useEffect(() => {
+      const requestOne = axios.get("/friends-list");
+      const requestTwo = axios.get("/poll");
```

Comment:
just best practice-wise, you're going to want to name these something better. There's nothing wrong with longer names: `friendsListRequest`, `pollRequest`

Reply:
removed this as one rejection causes issues